synthtool/gcp/common.py

```diff
@@ -92,6 +92,11 @@ def node_library(self, **kwargs) -> Path:
 
         kwargs["metadata"] = node.template_metadata()
         kwargs["publish_token"] = node.get_publish_token(kwargs["metadata"]["name"])
+        isGen = node.generate_index_ts(
```

Comment:
rename `isGen` to `generated_index_ts`

Reply:
After discussion, I refactor gnerate_index_tx -> None. 
isGen no longer use here.
Thanks.